package/gluon-scheduled-domain-switch/luasrc/usr/bin/gluon-switch-domain

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+local unistd = require 'posix.unistd'
+local util = require 'gluon.util'
+local site = require 'gluon.site'
+
+-- Returns true if node was offline long enough to perform domain switch
+function switch_after_min_reached()
+	if not unistd.access("/tmp/gluon_offline") then
+		return false
+	end
+
+	local current_epoch = os.time()
```

Comment:
Just call the variable *current_time*?

Reply:
I#ve dropped it altogether as it isn't used in this function.